src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
```

Comment:
use either number or text values for font-face through your CSS, but not both at the same time
it will make it easier to read your styles